package.json

```diff
@@ -4,6 +4,17 @@
   "description": "JSON-Schema + Faker",
   "homepage": "http://json-schema-faker.js.org",
   "main": "lib/index.js",
+  "scripts": {
```

Comment:
That horrible frankenstein called `grunt-parts` was a former way of thinking about automating... sorry for that.

From now on I'll try to use npm-scripts for everything.

Please give it a try. :+1: